### ISSUE TITLE
something is funky with the 1950 line in the no lag triple diff

### ISSUE THREAD

**Issue by sherryg1** [gliedgroup/ComparativeMortality#7]
It looks like a straight line at 0.  Maybe it is the left out category?  Probably best to have 1940 be left out and then just not show it.
Otherwise looking good and nice to see as stata graphs.  Work on the cover note for Adriana as well.

**Comment by benniezhu** [comment]
Yes it is a straight line at 0 because it is the left out category. The reason why I picked 1950 instead of 1940 was so that both the lag and no lag models would have the same base category; this is also why I chose 10-19 as the base age category as well.

**Comment by benniezhu** [comment]
Do you think Adrianna would strongly prefer having the figures included in a separate powerpoint or would sending her a memo that points to things in the github be ok?

**Comment by sherryg1** [comment]
If she can access the GitHub that’s fine

Sherry Glied

**Comment by benniezhu** [comment]
I just added a cover memo. It can be found in the pub folder; do you think I should revise the figures, remove those lines and standardize the Y axis a little better, before sending it to her?

**Comment by benniezhu** [comment]
Fixed the graphs so that they dont have flat lines for the base categories
